src/Http/Role/AddLabelToRoleRequestHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\AddLabel;
+use Fig\Http\Message\StatusCodeInterface;
+use InvalidArgumentException;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Response;
+
+class AddLabelToRoleRequestHandler implements RequestHandlerInterface
+{
+    private CommandBus $commandBus;
+
+    private ReadRepositoryInterface $labelRepository;
+
+    public function __construct(CommandBus $commandBus, ReadRepositoryInterface $labelRepository)
+    {
+        $this->commandBus = $commandBus;
+        $this->labelRepository = $labelRepository;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
```

Comment:
A lot of the `handle` is duplicated inside `RemoveLabelFromRoleRequestHandler`, maybe a trait can help?

Reply:
Extracted getLabelId logic in a trait. The roleId is refactored in part 3